src/plugins/intel_gpu/include/intel_gpu/runtime/format.hpp

```diff
@@ -206,6 +206,7 @@ struct format {
         os_i_osv16__ai8,                              ///< format used only for fully connected weights
         os_i_osv8__ai8,                               ///< format used only for fully connected weights
 
+        goix,                                         ///< format used for weights for 1D convolution
```

Comment:
Was your original target issue using grouped format?

Reply:
As this PR tried to fix two issues. And they were independent; when the 1st issue was fixed, the 2nd issue was produced. The 1st was about weights_reorder, and the 2nd issue was goix was not supported.
And goix format was added.